package.json

```diff
@@ -51,6 +59,7 @@
     "react": "^16.8.2 | ^17.0.0 | ^18.0.0"
   },
   "dependencies": {
-    "@babel/runtime-corejs2": "^7.17.8"
+    "@babel/runtime-corejs2": "^7.17.8",
+    "typescript": "^4.7.4"
```

Comment:
Typescript should be under `devDependencies`

Reply:
Good catch